client-side/src/components/report.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import GenericButton from '../stories/Button/GenericButton';
+import DateInputs from "../stories/DateTime/DateInputs";
+const DateTimePicker = () => {
+    const [startDate, setStartDate] = useState('');
+    const [endDate, setEndDate] = useState('');
+    const storeArr = () => {
+            const dataArr = [startDate, endDate];
+             //fillData(dataArr);
+       
+    }
+       
+
+    const handleStartDateChange = (start) => {
+        console.log('Start Date Selected:', start);
+        setStartDate(start);
+    };
+    const handleEndDateChange = (end) => {
+        console.log('End Date Selected:', end);
+        setEndDate(end);
+    };
+    const isButtonDisabled = (!startDate || !endDate) ||startDate > endDate; // Button is disabled when startDate or endDate is empty
+    console.log('Is Button Disabled:', isButtonDisabled);
+
+    
+    return (
+        <div>
+            <DateInputs onChange={handleStartDateChange} className="start" />
```

Comment:
I need to return 2 dates because this is a function that is activated when the user requests to see his table between certain dates. I made separately in the previous task a generic component of one date and used it here.